storage/innobase/btr/btr0scrub.cc

```diff
@@ -277,88 +265,74 @@ btr_page_needs_scrubbing(
 
 	} else if (allocated == BTR_SCRUB_PAGE_FREE ||
 		   allocated == BTR_SCRUB_PAGE_ALLOCATION_UNKNOWN) {
-
 		switch (fil_page_get_type(page)) {
-		case FIL_PAGE_INDEX:
-		case FIL_PAGE_TYPE_ZBLOB:
-		case FIL_PAGE_TYPE_ZBLOB2:
-			break;
-		default:
-			/**
+			case FIL_PAGE_INDEX:
+			case FIL_PAGE_TYPE_ZBLOB:
+			case FIL_PAGE_TYPE_ZBLOB2:
+				break;
+			default:
+				/**
 			* If this is a dropped page, we also need to scrub
 			* BLOB pages
 			*/
 
-			/* if encountering page type not needing scrubbing
+				/* if encountering page type not needing scrubbing
 			release reference to table object */
-			return BTR_SCRUB_SKIP_PAGE_AND_CLOSE_TABLE;
+				return BTR_SCRUB_SKIP_PAGE_AND_CLOSE_TABLE;
 		}
 	}
 
-	if (block->page.id.space() == TRX_SYS_SPACE
-	    && btr_page_get_index_id(page) == IBUF_INDEX_ID) {
+	if (block->page.id.space() == TRX_SYS_SPACE &&
+	    btr_page_get_index_id(page) == IBUF_INDEX_ID) {
 		/* skip ibuf */
 		return BTR_SCRUB_SKIP_PAGE_AND_CLOSE_TABLE;
 	}
 
 	return BTR_SCRUB_PAGE;
 }
 
-/****************************************************************
-Handle a skipped page
-*/
-
+/** Handle a skipped page
+@param[in]	scrub_data
+@param[in]	needs_scrubbing */
```

Comment:
Lost comments on move
```suggestion
@param[in]	needs_scrubbing   return code from btr_page_needs_scrubbing */
```